tools/src/launcher/Main.hs

```diff
@@ -488,13 +505,11 @@ spawnNode (path, args, mbLogPath) = do
             createDirectoryIfMissing True (directory lp)
             (lp,) <$> openFile lp AppendMode
         Nothing -> do
-            tempdir <- fromString <$> getTemporaryDirectory
-            -- FIXME (adinapoli): `Shell` from `turtle` was giving us no-resource-leak guarantees
-            -- via the `Managed` monad, which is something we have lost here, and we are back to manual
-            -- resource control. In this case, however, shall we really want to nuke the file? It seems
-            -- something useful to have lying around in the filesystem. We should probably close the
-            -- `Handle`, though, but if this program is short lived it won't matter anyway.
-            IO.openTempFile tempdir "cardano-node-output.log"
+            (_, stdO, stdE, pid) <- runInteractiveProcess path (map toString args) Nothing Nothing
```

Comment:
dunno why you didn't use similar approach https://github.com/input-output-hk/cardano-sl/pull/1629/files#diff-6a553d1d73e7c27fa55aff67ff05a784R544.

it seems to be a duplication of code https://github.com/input-output-hk/cardano-sl/pull/1629/files#diff-6a553d1d73e7c27fa55aff67ff05a784R622

Reply:
Kinda hard to guess what you mean, the links point to the same thing so I am little confused. Do you mean that `runInteractiveProcess` should of should NOT be used ?
Anyway I abstracted out the `ProcessCreate` generation so I think it looks better now ?

Reply:
The code was a little bit changed, but why don't you use `createProcInherit`?
we have other levels of logging apart from `Info` which can be in stdout, but you write all logs from stdout as `Info`. I expect inheritance will save logging level

Reply:
I used inherited process handle and indeed I see more logging similar to other nodes